airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py

```diff
@@ -53,15 +53,6 @@ class Config:
         order=1,
     )
 
-    report_granularity: str = Field(
-        title="Report Aggregation Granularity *",
-        description="The granularity used for aggregating performance data in reports. See <a "
-        'href="https://docs.airbyte.com/integrations/sources/tiktok-marketing/#report-aggregation">the docs</a>.',
-        default=ReportGranularity.default().value,
-        enum=[g.value for g in ReportGranularity],
-        order=2,
-    )
-
```

Comment:
Does you spec has 'additionalProperties': true ?

Reply:
agreed, @midavadim we should confirm this before merging

Reply:
granularity option is added back.

We have tests with all possible configs - old, oauth, with granularity and without.